float32 x
float32 y
float32 phi
int32 seq_k
int32 seq_aruco